addons/binding/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/handler/ExecHandler.java

```diff
@@ -80,16 +80,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     }
                 }
             } else if (channelUID.getId().equals(INPUT)) {
-                if (command instanceof StringType) {
-                    String previousInput = lastInput;
-                    lastInput = command.toString();
-                    if (lastInput != null && !lastInput.equals(previousInput)) {
-                        if (getConfig().get(AUTORUN) != null && ((Boolean) getConfig().get(AUTORUN)).booleanValue()) {
-                            lastInput = command.toString();
-                            logger.trace("Executing command '{}' after a change of the input channel to '{}'",
-                                    getConfig().get(COMMAND), command.toString());
-                            scheduler.schedule(periodicExecutionRunnable, 0, TimeUnit.SECONDS);
-                        }
+                logger.error("command = {}", command.toString());
```

Comment:
I think you forgot to remove this logging to error.

Reply:
I removed that line of code. An updated JAR file is located here,

[org.openhab.binding.exec-2.2.0-SNAPSHOT.jar.zip](https://github.com/openhab/openhab2-addons/files/1282574/org.openhab.binding.exec-2.2.0-SNAPSHOT.jar.zip)